docs/index.rst

```diff
@@ -51,10 +54,103 @@ and high-level quantum programs. :ref:`Read more... <bloqs_library>`
+:ref:`API Reference<reference>`.
 
 .. toctree::
    :hidden:
 
    reference/index.rst
+
+
+Installation
+------------
+
+Basics
+~~~~~~
+
+Qualtran is a Python package. If you already have a working Python environment,
+
+.. code-block:: sh
+
+    pip install qualtran
+
+will install the library and its dependencies. Qualtran uses `Graphviz <https://graphviz.org/>`_
+to draw diagrams, which can't be installed using the Python package manager ``pip``. Follow
+the `Graphviz installation instructions <https://graphviz.org/download/>`_ to set it up.
+Qualtran requires Python 3.10 or newer.
+
+Installing from source
+~~~~~~~~~~~~~~~~~~~~~~
+
+Source code is hosted on `GitHub <https://github.com/quantumlib/Qualtran>`_. You may wish
+to inspect the source code for the bloqs provided in the library or interactively execute
+the Jupyter notebooks included in the repository. In this case, you can clone the git repository
+and install from source.
+
+.. code-block:: sh
+
+    git clone https://github.com/quantumlib/Qualtran.git
+    cd Qualtran/
+    pip install -e .
+
+If you plan on developing Qualtran or you would just like to install all optional and developer
+dependencies, you can run
+
+.. code-block:: sh
+
+    pip install -r dev_tools/requirements/envs/dev.env.txt
+
+from the repository root.
+
+
+Setting up a Python environment
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+There are many ways to install Python and Qualtran's dependencies. This section will provide
+a walkthrough of one particular way.
+
+First, download and install `Miniconda <https://docs.anaconda.com/free/miniconda/>`_. This is
+a small tool that lets you install and manage Python environments. It supports different
+versions of Python in each environment, and can be installed without Administrator privileges.
+After following the installation instructions for Miniconda on your platform, you can create
+and activate an environment for Qualtran:
+
+.. code-block:: sh
+
+    conda create -n my-qualtran-env python=3.11
+    conda activate my-qualtran-env
+
+That code snippet will create a new environment with the given name (choose whatever you like)
+and install Python into it. Each time you start a new terminal you must first activate
+your environment
+
+.. code-block:: sh
+
+    conda activate my-qualtran-env
+
+You can use conda directly to install Graphviz:
+
+.. code-block:: sh
+
+    conda install graphviz
+
+At this point, you have an isolated, modern Python installation with Graphviz installed. From here,
+you can follow the basic installation instructions.
+
+.. code-block:: sh
+
+    pip install qualtran
+
+Next Steps
+~~~~~~~~~~
+
+You should be able to import ``qualtran`` from a Python command prompt.
+
+.. code-block:: python
+
+    import qualtran
+
+If this is successful,
+you can move on to learning how to `write bloqs <./_infra/Bloqs-Tutorial.html>`_ or investigate
+the :ref:`bloqs library<bloqs_library>`.
```

Comment:
This link doesn't render?

Reply:
I was being slow. Thought it was a README

Reply:
this will be on the front page of the readthedocs "below the fold". It's restructured text which is the weirdest markup syntax but it's the native sphinx form